src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -256,6 +256,14 @@ typeSpec[boolean addImagNode]
     | builtInTypeSpec[addImagNode]
     ;
 
+// A type specification for a variable length parameter is a type name with
+// possible brackets afterwards that can end with an annotation.
+variableLengthParameterTypeSpec
```

Comment:
reference: [`LastFormalParameter`](https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-LastFormalParameter) which allows an optional `Annotation` just before the `...`